mod_reforged/hooks/skills/effects/adrenaline_effect.nut

```diff
@@ -0,0 +1,37 @@
+::Reforged.HooksMod.hook("scripts/skills/effects/adrenaline_effect", function(q) {
+	q.m.GrantsInjuryImmunity <- true;
+
+	q.create = @(__original) function()
+	{
+		__original();
+
+		this.m.Order = ::Const.SkillOrder.PermanentInjury - 100;	// This effect needs to flip the IsAffectedByInjuries flag before any of the injuries had the time to inflict their effects
+	}
+
+	q.getTooltip = @(__original) function()
+	{
+		local ret = __original();
+
+		if (this.m.GrantsInjuryImmunity)
+		{
+			ret.push({
+				id = 11,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = "Ignore the effects of non-permanent injuries"
```

Comment:
`::Reforged.Mod.Tooltips.parseString("Ignore the effects of [temporary injuries|Concept.InjuryTemporary]")`